docs/config/supernode_properties.md

```diff
@@ -0,0 +1,72 @@
+| failureCountLimit | 5 | when dfget client fails to finish distribution task up to failureCountLimit, supernode will add it to blacklist|
+| linkLimit | 20 * rate.MB | LinkLimit is set for supernode to limit every piece download network speed |
+| systemReservedBandwidth | 20 * rate.MB |  network rate reserved for system |
+| maxBandwidth | 200 * rate.MB | network rate that supernode can use |
+| enableProfiler | false | profiler sets whether supernode HTTP server setups profiler |
+| debug | false | switch daemon log level to DEBUG mode |
+| failAccessInterval | 3m0s | fail access interval is the interval time after failed to access the URL |
+| gcInitialDelay | 6s | gc initial delay is the delay time from the start to the first GC execution |
+| gcMetaInterval | 2m0s | gc meta interval is the interval time to execute the GC meta |
+| taskExpireTime | 3m0s | task expire time is the time that a task is treated expired if the task is not accessed within the time |
+| peerGCDelay | 3m0s | peer gc delay is the delay time to execute the GC after the peer has reported the offline |
+| gcDiskInterval | 15s | GCDiskInterval is the interval time to execute GC disk |
+| youngGCThreshold | 100GB | if the available disk space is more than YoungGCThreshold and there is no need to GC disk |
+| fullGCThreshold | 5GB | if the available disk space is less than FullGCThreshold and the supernode should gc all task files which are not being used |
+| IntervalThreshold | 2h0m0s | IntervalThreshold is the threshold of the interval at which the task file is accessed |
+
+### Some common configurations
+
+We use `--config` to specify the configuration file directory, the default value is `/etc/dragonfly/supernode.yml`
+In Dragonfly, supernode provides `listenPort` for dfgets to connect, and dfget downloads document from `downloadPort` instead of `listenPort`.
+We can also configure the supernode's IP via `advertiseIP`.
+
+### About gc parameters
+
+In supernode, gc will begin `gcInitialDelay` time after supernode works.
+Then supernode will run peer-gc goroutine and task-gc goroutine every  `gcMetaInterval` time.
+If a task isn't accessed by dfgets in `taskExpireTime` time, task-gc goroutine will gc this task.
+If a peer reports that it's offline and can't provide download service to other peers, peer-gc goroutine will gc this peer after `peerGCDelay` time.
+
+## Examples
+
+Parameters are configured in `/etc/dragonfly/supernode.yml`.
+
+```yaml
+base:
+  listenPort: 8002
+  downloadPort: 8001
+  advertiseIP: 192.168.50.156
+  failAccessInterval: 3m0s
+  gcInitialDelay: 6s
+  gcMetaInterval: 2m0s
+  taskExpireTime: 2m0s
+  peerGCDelay: 3m0s
+  gcDiskInterval: 15s
+```
+
+When deploying with Docker, you can mount the default path when starting up supernode image with `-v`.
+
+```sh
+docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 -v /etc/dragonfly:/etc/dragonfly dragonflyoss/supernode:0.4.3
```

Comment:
I think only mount the config file `/etc/dragonfly/supernode.yml` is enough. WDYT?